fr/reporting/upgrade.md

```diff
@@ -18,14 +18,12 @@ La montée de version de Centreon MBI se fait en 3 étapes :
 
 ## Monter de version du paquet
 
-Lors d'une montée de version mineure ou majeure (ex: 19.10.x à 20.04.x) il faut en premier lieu mettre à jour
- le dépôt contenant les paquets. Si vous n'avez pas encore ce nouveau dépôt, le demander au support Centreon.
+Lors d'une montée de version majeure (ex: 20.10.x à 21.04.x) il faut en premier lieu mettre à jour
```

Comment:
It seems to me we're not giving the same instructions in FR and EN:
"When you upgrade from a previous version to 21.04.x"
VS
"Lors d'une montée de version majeure (ex: 20.10.x à 21.04.x)"

Reply:
You're almost right, it's not precise enough.
We can replace the EN with "previous major version".